### DISCUSSION TITLE
Error login by token

### DISCUSSION THREAD

**Discussion by andytorres** [deta/cloud-docs#430] [Help] [answered]
Hi, I've read the [docs](https://docs.deta.sh/docs/cli/auth/#deta-access-tokens) and tried both ways: tokens file and env variable. Still, every time I try `deta login`, I got redirected to the web login. What could I be doing wrong?
Thanks in advance

**Comment by abdelhai** [comment, marked as answer]
`deta login` will open the login page because that is what it is supposed to do. you might be logged in already. try using the cli to see if it works.
